ray_provider/operators/ray.py

```diff
@@ -274,41 +162,118 @@ def __init__(
         self.num_gpus = num_gpus
         self.memory = memory
         self.ray_resources = resources
+        self.ray_cluster_yaml = ray_cluster_yaml
+        self.update_if_exists = update_if_exists
+        self.kuberay_version = kuberay_version
+        self.gpu_device_plugin_yaml = gpu_device_plugin_yaml
         self.fetch_logs = fetch_logs
         self.wait_for_completion = wait_for_completion
         self.job_timeout_seconds = job_timeout_seconds
         self.poll_interval = poll_interval
         self.xcom_task_key = xcom_task_key
         self.dashboard_url: str | None = None
         self.job_id = ""
-        self.terminal_state = {JobStatus.SUCCEEDED, JobStatus.STOPPED, JobStatus.FAILED}
+        self.terminal_states = {JobStatus.SUCCEEDED, JobStatus.STOPPED, JobStatus.FAILED}
```

Comment:
In theory this is also a breaking change and we should mention it in release notes

Reply:
This is not an input parameter that is being updated. Its a variable set within the code and we're just changing the name a little bit.